app/views/notes/_comments.html.erb

```diff
@@ -2,11 +2,11 @@
 
   <div id="comments" class="col-md-10 comments">
 
-    <h3><span id="comment-count"><%= @node.drupal_comments.length %></span> <%= t('notes._comments.comments') %></h3>
+    <h3><span id="comment-count"><%= @node.comments.length %></span> <%= t('notes._comments.comments') %></h3>
  
     <div id="comments-container">
-      <% @node.drupal_comments.each do |comment| %>
-        <% if comment.cid == @node.drupal_comments.last.cid %><a id="last" name="last"></a><% end %>
+      <% @node.comments.each do |comment| %>
```

Comment:
Here I guess we'd want to reverse the order with a `.order('timestamp ASC')` since that's no longer automatic.